openproblems/data/pancreas.py

```diff
@@ -52,36 +53,15 @@ def load_pancreas(test=False, integer_only=False):
         scprep.io.download.download_url(URL, filepath)
         adata = sc.read(filepath)
 
+    if keep_techs is not None:
+        adata = adata[adata.obs["tech"].isin(keep_techs)].copy()
+
     # NOTE: adata.X contains log-normalized data, so we're moving it
     adata.layers["log_normalized"] = adata.X
     adata.X = adata.layers["counts"]
     del adata.layers["counts"]
 
-    if integer_only:
-        adata = _get_pancreas_integer(adata)
-
     # Ensure there are no cells or genes with 0 counts
     utils.filter_genes_cells(adata)
 
     return adata
-
-
-def _get_pancreas_integer(adata: ad.AnnData):
```

Comment:
This was added only for spatial deconvolution and is now unused since we only take one batch.